### PR TITLE
[UIO-16]-Footer component

### PR DESCRIPTION
**Pull request by Hollister009**
https://issues.griddynamics.net/browse/UIO-16

### CODE REVIEW

File: src/app/components/Footer/Footer.js [hunk 1]
```diff
@@ -1,57 +1,40 @@
...
+import appConfig from '../../../config/appConfig';
+import './Footer.scss';
 
 const Footer = props => {
   const { headerFooterVisibility } = props;
   const visible = headerFooterVisibility.value;
+  const renderLinks = arr =>
+    arr.map(link => (
+      <li key={link.id}>
+        <a href={link.href} rel="noopener noreferrer" target="_blank">
```
Comment (andriytereschuk) [inline src/app/components/Footer/Footer.js]: Now we handle not origin links only. What about app links?
We should have a possibility to render router links.
eg: PLP link

File: src/app/components/Footer/Footer.js [hunk 2]
```diff
@@ -1,57 +1,56 @@
...
   const visible = headerFooterVisibility.value;
 
+  const renderLinks = list =>
+    list.map(link => {
+      const { id, value, href, isOrigin } = link;
+      return isOrigin ? (
+        <li key={id}>
+          <a href={href} rel="noopener noreferrer" target="_blank">
+            &bull;
+            {` ${value}`}
```
Comment (andriytereschuk) [inline src/app/components/Footer/Footer.js]: change to &bull;&nbsp;{value}

File: src/config/appConfig.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+      { id: 23, value: 'our products', href: '/products', isOrigin: false },
+      { id: 36, value: 'join us', href: '/', isOrigin: false },
+      { id: 11, value: 'contact us', href: '/', isOrigin: false },
+      { id: 24, value: 'terms and conditions', href: '/', isOrigin: false },
+      { id: 37, value: 'live support', href: '/', isOrigin: false },
+      { id: 12, value: 'support', href: '/', isOrigin: false },
+      { id: 25, value: 'our privacy', href: '/', isOrigin: false }
+    ],
+    payments: [
+      { id: 8, value: 'amex', href: 'https://www.americanexpress.com/', isOrigin: true },
```
Comment (andriytereschuk) [inline src/config/appConfig.js]: `isOrigin` means the same origin, but here is external link

File: src/app/components/Footer/Footer.js [hunk 4]
```diff
@@ -1,57 +1,56 @@
...
+import appConfig from '../../../config/appConfig';
+import './Footer.scss';
 
 const Footer = props => {
   const { headerFooterVisibility } = props;
   const visible = headerFooterVisibility.value;
 
+  const renderLinks = list =>
+    list.map(link => {
+      const { id, value, href, isOrigin } = link;
```
Comment (andriytereschuk) [inline src/app/components/Footer/Footer.js]: plz, set up next eslint rule: after `const` should be a blank line

File: src/app/components/Footer/Footer.js (hunk 2 above)
Comment (Hollister009) [reply src/app/components/Footer/Footer.js]: Alright

File: src/config/appConfig.js (hunk 3 above)
Comment (Hollister009) [reply src/config/appConfig.js]: Go it

File: src/app/components/Footer/Footer.js [hunk 5]
```diff
@@ -1,57 +1,56 @@
...
+      return isOrigin ? (
```
Comment (andriytereschuk) [inline src/app/components/Footer/Footer.js]: return isOrigin ? renderRouteLink(link) : renderExternalLink(link);

I think it's more declarative

File: src/app/components/Footer/Footer.js (hunk 4 above)
Comment (andriytereschuk) [reply src/app/components/Footer/Footer.js]: thanks

**Pull request closed without merging**